lib/std/os/windows/bits.zig

```diff
@@ -236,7 +236,11 @@ pub const FILE_NAME_INFORMATION = extern struct {
 };
 
 pub const IO_STATUS_BLOCK = extern struct {
-    Status: usize,
+    // "DUMMYUNIONNAME" expands to "u"
+    u: extern union {
+        Status: NTSTATUS,
+        Pointer: ?*c_void,
+    },
```

Comment:
Is this really better than status quo? `@ptrCast` from a `?*c_void` is not much different than `@intToPtr` from a `usize`.

Reply:
Status isn't a `usize`, it's a `u32`/`DWORD`.